templates/apiMultipartHelper.hbs

```diff
@@ -0,0 +1,17 @@
+import { Request, Response, NextFunction } from 'express'
+import multer from 'multer'
+import { Api } from '../../models'
+
+export const upload = multer({storage: multer.memoryStorage()})
```

Comment:
I wonder whether we might want to just have a single `multer` for the whole application?